app/config/services.yml

```diff
@@ -120,4 +120,4 @@ services:
 
     pelagos.util.search:
         class: Pelagos\Util\Search
-        arguments: ["@fos_elastica.finder.search_pelagos.dataset"]
\ No newline at end of file
+        arguments: ["@fos_elastica.finder.search_pelagos.dataset", "@doctrine.orm.entity_manager"]
```

Comment:
Add newline please. We don't like to see those "not-enter" icons.

Reply:
Done 👍